src/CMakeLists.txt

```diff
@@ -164,30 +165,30 @@ add_sanitizers(digest)
 install(
     PROGRAMS ${mksquashfs_INSTALL_DIR}/mksquashfs ${CMAKE_CURRENT_BINARY_DIR}/runtime
     DESTINATION bin
+    COMPONENT applications
 )
 
 install(
     TARGETS AppRun appimaged appimagetool digest validate
-    RUNTIME DESTINATION bin
-    LIBRARY DESTINATION lib
-    ARCHIVE DESTINATION lib/static
-    INCLUDES DESTINATION include
+    RUNTIME DESTINATION bin COMPONENT applications
+    LIBRARY DESTINATION lib COMPONENT applications
+    ARCHIVE DESTINATION lib/static COMPONENT applications
```

Comment:
We should perhaps change this to `lib`, too. However, when using `find_library` as suggested in the CMake module config, it would be found even if built statically.

Reply:
I guess that we would need to add another target for the static lib.

Reply:
I'm not saying we should build a static library. You just mentioned another cross distro issue, which is library locations. Debian based systems put both `.a` and `.so` files in `<prefix>/lib`, whereas other distros might put it in a `static` subdirectory. Since we're focusing on building for Debian, I thought the change might be a good idea. But I let this decision to you. I just wanted to express my point on it.